tools/findFilesThatContainImages.sh

```diff
@@ -0,0 +1,26 @@
+doit(){
```

Comment:
https://stackoverflow.com/questions/8967902/why-do-you-need-to-put-bin-bash-at-the-beginning-of-a-script-file

```suggestion
#!/bin/bash
doit(){
```